src/types/api/resources/DisputesResource.ts

```diff
@@ -98,16 +152,22 @@ export interface components {
             /** @description Payment method type code of the transaction, e.g. 'klarna', 'visa', 'mc' */
             type: string;
         };
-        Link: {
-            /** @description Cursor for a different page */
-            cursor: string;
+        DisputeListItem: {
+            amount: components['schemas']['Amount'];
+            /** Format: date-time */
+            createdAt: string;
+            disputePspReference: string;
+            /** Format: date-time */
+            dueDate?: string;
+            paymentMethod: components['schemas']['PaymentMethod'];
+            reason: components['schemas']['DisputeReason'];
+            status: components['schemas']['DisputeStatus'];
         };
-        Links: {
-            /** @description Link to a different page */
-            next: components['schemas']['Link'];
-            /** @description Link to a different page */
-            prev: components['schemas']['Link'];
+        DisputeListResponse: {
+            disputes?: components['schemas']['DisputeListItem'][];
```

Comment:
This type is missing the structure for the disputes list endpoint as before. I think it should look more like this:
```suggestion
        DisputeListResponse: {
            _links?: components['schemas']['Links'];
            data?: components['schemas']['DisputeListItem'][];
```

Reply:
✅